scripts/ci/check_compliance.py

```diff
@@ -1018,7 +1019,7 @@ def check_no_undef_outside_kconfig(self, kconf):
         "BOARD_", # Used as regex in scripts/utils/board_v1_to_v2.py
         "BOARD_MPS2_AN521_CPUTEST", # Used for board and SoC extension feature tests
         "BOARD_NATIVE_SIM_NATIVE_64_TWO", # Used for board and SoC extension feature tests
-        "BOARD_NATIVE_SIM_NATIVE_ONE", # Used for board and SoC extension feature tests
+        "BOARD_NATIVE_SIM_NATIVE_ONE",  # Used for board and SoC extension feature tests
```

Comment:
Unrelated change?

Reply:
Probably some auto-formatting thing.

Reply:
Fixed!

Reply:
I might not have pushed to my branch yet. Weird.